l2-bridge-ethereum/src/constants.ts

```diff
@@ -107,6 +107,8 @@ export const ARBITRUM_GATEWAY_SET_EVENT =
 export const LINEA_CUSTOM_CONTRACT_SET_EVENT =
   "event CustomContractSet(address indexed nativeToken, address indexed customContract, address indexed setBy);";
 
+export const BSC_CROSS_CHAIN_CONTROLLER = "";
```

Comment:
Please add L1 to the name for clarity
```suggestion
export const BSC_L1_CROSS_CHAIN_CONTROLLER = "";
```

Reply:
Good catch, thanks. Addressed in [63574c0](https://github.com/lidofinance/alerting-forta/pull/564/commits/63574c096df8db69e1203cfe6b200e01d6169f31).